src/TestSuite/MiddlewareDispatcher.php

```diff
@@ -69,7 +68,7 @@ public function __construct($test, $class = null, $constructorArgs = null)
     /**
      * Run a request and get the response.
      *
-     * @param \Cake\Http\ServerRequest $request The request to execute.
+     * @param \Psr\Http\Message\RequestInterface $request The request to execute.
```

Comment:
Should be `ServerRequestInterface`

Reply:
The `$server->run($request);` at line 95 returns `\Psr\Http\Message\RequestInterface`.

If I change to `SeverRequestInterface` I will likely get a return type mismatch warning.

Reply:
`Psr\Http\Message\RequestInterface` doesn't exist :slightly_smiling_face:. So it's probably a typo at some place.

Reply:
> The $server->run($request); at line 95 returns \Psr\Http\Message\RequestInterface.

`Server::run()` returns a response of type `\Psr\Http\Message\ResponseInterface`, not request.

Reply:
Argh... I was looking at the return type but the comment was about the 1st param. Sorry. Fixing now.